satellite/accounting/live/cache.go

```diff
@@ -21,13 +21,13 @@ var (
 
 // Config contains configurable values for the live accounting service.
 type Config struct {
-	StorageBackend string `help:"what to use for storing real-time accounting data" default:"memory"`
+	DB string `help:"what to use for storing real-time accounting data" default:"memory"`
```

Comment:
do we want to change this config name? we will need to change it for all the production satellites, etc